src/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -80,20 +80,28 @@ public partial class HttpClientHandler : System.Net.Http.HttpMessageHandler
         public HttpClientHandler() { }
         public bool AllowAutoRedirect { get { throw null; } set { } }
         public System.Net.DecompressionMethods AutomaticDecompression { get { throw null; } set { } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
>By doing this we are introducing some APIs that aren't going to be
compatible with .NET 4.6.1. If they are used they will throw
MissingType or MissingMethod exceptions. We chose to be back compatible
with netstandard1.x over 100% compatible with .NET 4.6.1. To see the
incompatible APIs refer to ApiCompatBaseline.net461.txt.
To help mitigate people using these APIs we have decided to mark them
with EditorBrowsableState.Never which will hide them in intellisense.

The only difference, though, is that these particular HttpClientHandler properties ARE supported on all platforms since we OOB System.Net.Http on Desktop and it is available across all other platforms that support netstandard20.

So, I would argue that even for the netstandard reference assembly itself, we shouldn't be adding these attributes either.